SPECS/sriov-network-device-plugin/sriov-network-device-plugin.spec

```diff
@@ -0,0 +1,35 @@
+Summary:        Plugin for discovering and advertising networking resources
+Name:           sriov-network-device-plugin
+Version:        3.3
```

Comment:
Do we have a need for this specific version? This version is almost two years old, and newer versions have a number of bugfixes and security fixes (with respect to vendored packages).

Reply:
version 3.4 is the one required for one of our platforms.

Reply:
Should I change and retest the spec and build with v3.3.2

Reply:
After discussing with Roaa, we have decided that v3.4.0 is the appropriate version. The spec was updated with this version and the rpm build was retested and successful with v3.4.0